src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/DependenciesGraphProvider.cs

```diff
@@ -152,7 +145,7 @@ Graph IGraphProvider.Schema
         {
             try
             {
-                await EnsureInitializedAsync().ConfigureAwait(false);
+                await InitializeAsync().ConfigureAwait(true);   // InitializeAsync switches to the UI thread, we want to get off it
```

Comment:
Hmm, need to figure out if this will actually work as I'm on background thread. I think maybe I need to just await the default scheduler.